src/components/NewPetForm.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+
+const NewPetForm = props => {
+  const [newPet, setNewPet] = useState({
+    name:"", 
+    phoneNumber:"",
+    email:"",
+    petName:"",
+    petAge:"",
+    petType:"",
+    petImage:"",
+    vaccinationStatus:""
+  })
+  const handlePetChange = event => {
+    setNewPet({
+      ...newPet,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+  const handlePetSubmit = event => {
+    event.preventDefault()
+    let payload = {
+      name: newPet.name,
+      phoneNumber: newPet.phoneNumber,
+      email: newPet.email,
+      petName: newPet.petName,
+      petAge: newPet.petAge,
+      petType: newPet.petType,
+      petImage: newPet.petImage,
+      vaccinationStatus: newPet.vaccinationStatus
+    }
+    //fetch post goes here?

```

Comment:
Again the comment here. I understand that you are creating this outside of a specific feature in the PR. Normally you would have both the fetch function and form logic in a single feature PR. Line 33, has the function name underscored out. I recommend adding that fetch function in, so that this PR works without errors.